packages/desktop/src/renderer/sagas/notifications/notifications.saga.ts

```diff
@@ -57,6 +62,16 @@ export function* displayMessageNotificationSaga(
     const sender = allUsers[message.pubKey]?.username
     if (!sender || sender === currentIdentity?.nickname) return
 
+    // Check is user info message
+    if (message.type === MessageType.Info && sender !== ownerNickname) {
```

Comment:
Why do we try to transform message before even checking notificationsConfig? I think it should be transformed after all initial checks (after `if (!action.payload.isVerified) return`)